features/measurements/impl/src/main/java/org/opennms/netmgt/measurements/impl/JEXLExpressionEngine.java

```diff
@@ -99,6 +99,9 @@ public void applyExpressions(final QueryRequest request, final FetchResults resu
         final Map<String, double[]> columns = results.getColumns();
         final int numRows = timestamps.length;
 
+        // Calculate the time span
+        jexlValues.put("__diff_time", numRows < 1 ? 0d : timestamps[numRows-1] - timestamps[0]);
```

Comment:
During my tests, I figured out that you have to divide the difference between the two timestamps by 1000 in order to get the same numbers VDEF provides.